internal/mode/static/state/graph/backend_tls_policy.go

```diff
@@ -0,0 +1,153 @@
+			Namespace: helpers.GetPointer((v1.Namespace)(gateway.Source.Namespace)),
+			Name:      v1.ObjectName(gateway.Source.Name),
+		}
+		var alreadyAncestor bool
+		for _, ancestor := range backendTLSPolicy.Status.Ancestors {
+			if string(ancestor.ControllerName) == ctlrName && ancestor.AncestorRef.Name == ancestorRef.Name &&
+				*ancestor.AncestorRef.Namespace == *ancestorRef.Namespace {
+				alreadyAncestor = true
+				break
+			}
+		}
+		if !alreadyAncestor {
+			valid = false
+			conds = append(conds, staticConds.NewBackendTLSPolicyIgnored("too many ancestors, cannot attach a new Gateway"))
+		}
+	}
+	if err := validateBackendTLSHostname(backendTLSPolicy); err != nil {
+		valid = false
+		conds = append(conds, staticConds.NewBackendTLSPolicyInvalid(fmt.Sprintf("invalid hostname: %s", err.Error())))
+	}
+	if backendTLSPolicy.Spec.TLS.CACertRefs != nil && len(backendTLSPolicy.Spec.TLS.CACertRefs) > 0 {
+		if err := validateBackendTLSCACertRef(backendTLSPolicy, configMapResolver); err != nil {
+			valid = false
+			conds = append(conds, staticConds.NewBackendTLSPolicyInvalid(
+				fmt.Sprintf("invalid CACertRef: %s", err.Error())))
+		} else {
+			caCertRef = types.NamespacedName{
+				Namespace: backendTLSPolicy.Namespace, Name: string(backendTLSPolicy.Spec.TLS.CACertRefs[0].Name),
+			}
+		}
+	} else if backendTLSPolicy.Spec.TLS.WellKnownCACerts != nil {
+		if err := validateBackendTLSWellKnownCACerts(backendTLSPolicy); err != nil {
+			valid = false
+			conds = append(conds, staticConds.NewBackendTLSPolicyInvalid(
+				fmt.Sprintf("invalid WellKnownCACerts: %s", err.Error())))
+		}
+	} else {
+		valid = false
+		conds = append(conds, staticConds.NewBackendTLSPolicyInvalid("CACertRefs and WellKnownCACerts are both nil"))
+	}
+	return valid, caCertRef, conds
+}
+
+func validateBackendTLSHostname(btp *v1alpha2.BackendTLSPolicy) error {
+	h := string(btp.Spec.TLS.Hostname)
+
+	if err := validateHostname(h); err != nil {
+		path := field.NewPath("tls.hostname")
+		valErr := field.Invalid(path, btp.Spec.TLS.Hostname, err.Error())
+		return valErr
+	}
+	return nil
+}
+
+func validateBackendTLSCACertRef(btp *v1alpha2.BackendTLSPolicy, configMapResolver *configMapResolver) error {
+	if len(btp.Spec.TLS.CACertRefs) != 1 {
+		path := field.NewPath("tls.cacertrefs")
+		valErr := field.TooMany(path, len(btp.Spec.TLS.CACertRefs), 1)
+		return valErr
+	}
+	if btp.Spec.TLS.CACertRefs[0].Kind != "ConfigMap" {
+		path := field.NewPath("tls.cacertrefs[0].kind")
+		valErr := field.NotSupported(path, btp.Spec.TLS.CACertRefs[0].Kind, []string{"ConfigMap"})
+		return valErr
+	}
+	if btp.Spec.TLS.CACertRefs[0].Group != "" && btp.Spec.TLS.CACertRefs[0].Group != "core" {
+		path := field.NewPath("tls.cacertrefs[0].group")
+		valErr := field.NotSupported(path, btp.Spec.TLS.CACertRefs[0].Group, []string{"", "core"})
+		return valErr
+	}
+	nsName := types.NamespacedName{Namespace: btp.Namespace, Name: string(btp.Spec.TLS.CACertRefs[0].Name)}
```

Comment:
We don't need to, because the CA cert reference is of type LocalObjectReference which [identifies an API object within the namespace of the referrer](https://pkg.go.dev/sigs.k8s.io/gateway-api@v1.0.0/apis/v1#LocalObjectReference) and doesn't have a Namespace field.